.github/workflows/config-jsonschema-check.yml

```diff
@@ -0,0 +1,125 @@
+name: config-jsonschema-check
```

Comment:
Nit: Almost of the workflows use `under_score` instead of `kebab-case`.
I suppose it's better to use it here too.

Reply:
Replaced with `_`

Reply:
I meant the the filename too

Reply:
Fixed.